python/pandemic_simulator/environment/pandemic_env.py

```diff
@@ -46,6 +47,36 @@ def __init__(self,
         :param sim_steps_per_regulation: number of sim_steps to run for each regulation
         :param non_essential_business_location_ids: an ordered list of non-essential business location ids
         """
+        
+        
+        
+
+        self.action_space=spaces.Discrete(2) 
+        '''
+        action=0 --> Reduce level of regulation.
+        action=1 --> Maintain level of regulation.
+        action=2 --> Increase level of regulation.
+        '''
+
+        #Uncomment below line to make actions to choose regulation level
+        #self.action_space = gym.spaces.Discrete(len(self._stage_to_regulation))
```

Comment:
I don't think we should be changing the default action space for the env here. Instead you can create a Gym's ActionWrapper `gym.core.ActionWrapper` and use that for your experiments. 
In the action wrapper you can change the action_space and the returned action. Let me know if you need more help to setup and use an action wrapper. You should find many examples in the internet.

Reply:
Do you mean the final action space will be [0,no. of regulations] and by using an action wrapper we should allow the actions to be -1,0,1. The action 0 to n is not allowed.
I was thinking maybe we can have two variations of the environment like PandemicSim and PandemicSimN, where PandemicSim can have action space [0,2] for -1,0,1 and actions the same and PandemicSimN can have action space and actions [0,n] similar to the ReversedAddition environment that used to be in OpenAI gym [link].(https://github.com/Rohan138/gym-algorithmic/blob/main/gym_algorithmic/reversed_addition.py)

Reply:
@varunrajk Could you give your thoughts on this?